src/blocks/block-notice/components/inspector.js

```diff
@@ -38,8 +38,8 @@ export default class Inspector extends Component {
 
 		// Notice dismiss options
 		const noticeDismissOptions = [
-			{ value: null, label: __( 'Always Show' ) },
-			{ value: 'ab-dismissable', label: __( 'Dismissable' ) },
+			{ value: null, label: __( 'Always Show', 'atomic-blocks' ) },
+			{ value: 'ab-dismissable', label: __( 'Dismissable', 'atomic-blocks' ) },
```

Comment:
Should be `Dismissible` and `ab-dismissible`. If that requires a deprecation, I'd vote to just change the label for now.